ui/components/app/terms-of-use-popup/terms-of-use-popup.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import TermsOfUsePopup from '.';
+
+export default {
+  title: 'Components/App/TermsOfUsePopup',
+};
+
+export const DefaultStory = () => <TermsOfUsePopup />;
+
```

Comment:
Since, `onAccept` is a required prop. We need to pass it in args else it will keep throwing errors like this

![Screenshot 2023-04-10 at 1 23 40 PM](https://user-images.githubusercontent.com/39872794/230854883-c8b9d100-f7cd-4c08-a7c9-1695c11b7a61.jpg)

```suggestion
export default {
  title: 'Components/App/TermsOfUsePopup',
  component: TermsOfUsePopup,
  argTypes: {
    onAccept: {
      action: 'onAccept',
    },
  },
};

export const DefaultStory = (args) => <TermsOfUsePopup {...args} />;

```